Modules/Quotas/src/Quotas.Domain/Aggregates/Relationships/Relationship.cs

```diff
@@ -1,9 +1,9 @@
 ﻿namespace Backbone.Modules.Quotas.Domain.Aggregates.Relationships;
 public class Relationship : ICreatedAt
 {
-    public string Id { get; set; }
-    public string From { get; set; }
-    public string To { get; set; }
+    public string? Id { get; set; }
+    public string? From { get; set; }
```

Comment:
Not sure but I think this should not be nullable either.

Reply:
👍